tests/languages/arturo/symbol_feature.test

```diff
@@ -0,0 +1,13 @@
+"hello" = " world"
```

Comment:
Could we please also test all other symbols defined by the regex? This would have caught the `---` bug.

Reply:
Done ;) (hopefully...)